dynamic_rest/fields/fields.py

```diff
@@ -118,8 +122,15 @@ def bind(self, *args, **kwargs):
         parent_model = getattr(self.parent.Meta, 'model', None)
 
         remote = is_field_remote(parent_model, self.source)
+
+        def get_model_field():
+            if DJANGO110:
+                return parent_model._meta.get_field(self.source)
+            else:
+                return parent_model._meta.get_field_by_name(self.source)[0]
```

Comment:
As you probably saw in #141, we need to support 1.7 for a while longer for our own purposes, but we'll likely drop support in the near future.